style.css

```diff
@@ -87,12 +89,16 @@ body {
 .icon {
     width: var(--icon-size);
     height: var(--icon-size);
-    /* text-shadow: 0px 2px 24px rgba(0,0,0,0.5); */
+    /* text-shadow: 0px 2px 24px rgba(0,0,0,0.5);  */
     -webkit-filter: var(--shadow);
     filter: var(--shadow);
     padding: var(--margin-normal);
 }
 
+#video-container :not(:last-child) {
```

Comment:
Denne selectoren var litt forvirrende. Etter å ha sett på det, ser det ut til å være alle divs inne i `#video-container`? Jeg forslår en av to: 
1) Legge til en ny klasse på de some bør ha dette
2) `:not(video)` som andre selector

Reply:
Selectoren velger alle divs inne i #video-container minus den siste som er der videoen er. Og css-en er z-index som løfter alle divene over videoen. Var et problem i chrome på mac at jeg ikke fikk trykket på ikonene fordi de ikke var på et lag over selvom jeg så ikonene ligge over videoen. Vet ikke.